src/sparseml/pytorch/image_classification/export.py

```diff
@@ -13,79 +13,77 @@
-                 [--model-tag MODEL_TAG] [--save-dir SAVE_DIR]
+Usage: sparseml.image_classification.export [OPTIONS]
+
+  SparseML-PyTorch Integration for exporting image classification models to
+  onnx along with sample inputs and outputs
+
+Options:
+  -d, --dataset TEXT              The dataset used for training, ex:
+                                  `imagenet`, `imagenette`, `cifar10`, etc.
+                                  Set to `imagefolder` for a generic dataset
+                                  setup with imagefolder type structure like
+                                  imagenet or loadable by a dataset in
+                                  `sparseml.pytorch.datasets`  [required]
+  --dataset-path, --dataset_path DIRECTORY
+                                  The root dir path where the dataset is
+                                  stored or should be downloaded to if
+                                  available  [required]
+  --checkpoint-path, --checkpoint_path TEXT
+                                  A path to a previous checkpoint to load the
+                                  state from and resume the state for
+                                  exporting
+  --arch_key, --arch-key TEXT     The architecture key for image
+                                  classification model; example: `resnet50`,
+                                  `mobilenet`. Note: Will be read from the
+                                  checkpoint if not specified
+  --num-samples, --num_samples INTEGER
+                                  The number of samples to export along with
+                                  the model onnx and pth files (sample inputs
+                                  and labels as well as the outputs from model
+                                  execution)  [default: 100]
+  --onnx-opset, --onnx_opset INTEGER
+                                  The onnx opset to use for exporting the
+                                  model  [default: 11]
+  -z, --use_zipfile_serialization_if_available,
+  --use-zipfile-serialization-if-available / -Z, --no_zipfile_serialization,
+  --no-zipfile-serialization
+                                  For torch >= 1.6.0 only exports the Module's
+                                  state dict using the new zipfile
+                                  serialization. Default is True, has no
+                                  effect with lower torch versions  [default:
+                                  use_zipfile_serialization_if_available]
+  --pretrained TEXT               The type of pretrained weights to use, loads
+                                  default pretrained weights for the model if
+                                  not specified or set to `True`. Otherwise
+                                  should be set to the desired weights type:
+                                  [base, optim, optim-perf]. To not load any
+                                  weights setto one of [none, false]
```

Comment:
nit: setto -> set to

Reply:
<img src="https://media0.giphy.com/media/spG6XJgqavFmFwhR54/giphy.gif"/>